components/vc/front/src/torque.c

```diff
@@ -218,6 +231,53 @@ static void evaluate_launch_control(float32_t accelerator_position, float32_t br
 #endif
 }
 
+static float32_t calc_torque_correction(float32_t target_slip, float32_t actual_slip, float32_t max_torque)
+{
+    epid_pi_calc(&torque_data.traction_control_pid, target_slip, actual_slip);
+    epid_pi_sum(&torque_data.traction_control_pid, 0.0f, max_torque);
+    epid_util_ilim(&torque_data.traction_control_pid, 0.0f, TRACTION_CONTROL_ILIM);
+
+    return torque_data.traction_control_pid.y_out;
+}
+
+static void evaluate_traction_control(void)
+{
+    CAN_digitalStatus_E traction_control_requested = CAN_DIGITALSTATUS_SNA;
+    float32_t rear_axle_rpm = 0.0f;
```

Comment:
I could have sworn we were camel case. Did I miss a memo?

Reply:
I think functions are camel, but variables are underscored? Atleast thats how I have always done it

Reply:
Damn it must be nice being the one to dictate these things jeez louise. Shouldve came to school first IG

jkjk, I can get behind whatever as long as we formalize it, would you want to make a doc with these rules as oyu have them internalized, say in like `docs/guidelines.md` folder. I started something there a while ago and never followed through. Maybe get oyur input on it and add whetever, then once it set we can just continue forwards doing the right thing